electron/renderer.js

```diff
@@ -3,7 +3,18 @@
 const {ipcRenderer} = require('electron');
 
 self.F = self.F || {};
-const ns = self.F.electron = {};
+const ns = self.F.electron = {
+    isElectron: true
```

Comment:
I think this is redundant, no?  presence of F.electron alone indicates that you're in it.